resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerConfigurationProperties.java

```diff
@@ -19,7 +19,6 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.Ordered;
 
-@Configuration
```

Comment:
I deleted this annotation. I don't know why Properties class has `Configuration` annotation.